### DISCUSSION TITLE
Export/import playlists

### DISCUSSION THREAD

**Discussion by ghost** [libre-tube/LibreTube#1303] [Q&A] [answered]
Hi dev i just wondering if you will add ability to import export playlist from yt just like newpipe did?🙏

Edit:
I think this is are useful since user might be want to be anonymous on YouTube 
Also is this issues already exist? Idk even after checking issues tab

**Comment by Bnyro** [comment, marked as answer]
Importing isn't easy yet due to api restrictions but planned for the future.

For exporting it'd be nice if you could provide a sample of an exported file so that it'll be compatible with NewPipe.

**Reply by ghost** [reply]
1. Yes i understand if need time the important thing is you have plan to add it at least
2. Format is csv taken from google takeout
You can go to google takeout and export only yt data
(playlist / subscriber)

Takeout.google.com

(If you still need formatting example i will send it privately so is there a way how i can contact you privately?)

@Bnyro

**Reply by Bnyro** [reply]
Since I don't own a Google account it would be nice if you could help me out with an example file :)

You can reach me on Matrix, @bnyro:matrix.org, or just find me by joining the LibreTube Matrix room mentioned in the README.

**Reply by ghost** [reply]
Got it 
File will sended soon 🙏

**Comment by silentTeee** [comment]
So for your own YouTube playlists, the Google Taekout method doesn't work yet, but there's a pretty simple alternative if you don't mind adding playlists one at a time. Here are the steps:

1. While in the official YouTube app and logged into your account, navigate to your playlists
2. Select the playlist that you want to migrate, and go to edit it
![Screenshot_20221208-085840~2.png](https://user-images.githubusercontent.com/20748060/206470088-2ed52bfe-339b-4087-83f5-535c3e8c1102.png)
3. Set the playlist privacy settings from "Private" to "Unlisted" (anyone with the link can view)
![Screenshot_20221208-085857.png](https://user-images.githubusercontent.com/20748060/206470601-e8709153-27ce-4e4b-8a2b-6ef89de42a65.png)
4. Back out to the view showing the playlist, and using the "Share" option, select the LibreTube app. This will open the playlist link in LibreTube
![Screenshot_20221208-085840~3.png](https://user-images.githubusercontent.com/20748060/206471376-0809a1d6-f2c5-4c87-a4ba-c52ca4e82487.png)
5. Using the 3-dot menu, select "Clone Playlist"
![Screenshot_20221208-093744~2.png](https://user-images.githubusercontent.com/20748060/206474969-1cfb0173-b96a-477f-9fe9-127aa2c3b717.png)
![Screenshot_20221208-093749~2.png](https://user-images.githubusercontent.com/20748060/206475017-8160a2ab-5162-4c70-9e58-7c2562591966.png)
6. You're done. ✌️

Edit to add: This method will also work for public playlists that you don't own but want to. If you simply want to keep an eye on a playlist that's owned by someone else and gets changed frequently, used the Bookmark feature. 

Edit 2: added more screenshots for clarity

**Reply by cesc1989** [reply]
Really useful! Thanks a bunch.

**Comment by whisper97** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
